opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/ListTaskPresenter.java

```diff
@@ -106,6 +114,8 @@
 
     private FeatureCollection featureCollection;
 
+    private List<Feature> filteredFeatureCollection;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'filteredFeatureCollection' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)